app/Http/Requests/UpdateAssetRequest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Models\Asset;
+use Illuminate\Support\Facades\Gate;
+use Illuminate\Validation\Rule;
+
+class UpdateAssetRequest extends ImageUploadRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return Gate::allows('update', new Asset);
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        $rules = array_merge([
+            'model_id' => ['integer', 'exists:models,id,deleted_at,NULL', 'not_array'],
+            'status_id' => ['integer', 'exists:status_labels,id'],
+            'asset_tag' => ['min:1', 'max:255', 'not_array', Rule::unique('assets', 'asset_tag')->ignore($this->asset)->withoutTrashed()],
+            'name' => ['nullable', 'max:255'],
+            'company_id' => ['nullable', 'integer', 'exists:companies,id'],
+            'warranty_months' => ['nullable', 'numeric', 'digits_between:0,240'],
+            'last_checkout' => ['nullable', 'date_format:Y-m-d H:i:s'],
+            'expected_checkin' => ['nullable', 'date'],
+            'last_audit_date' => 'nullable|date_format:Y-m-d H:i:s',
```

Comment:
Notwithstanding my comment a few lines up, the rest of these rules are specified as arrays, but these two are strings. Is that on purpose?